packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -37,20 +36,19 @@ const String _extensionMethod = 'ext.flutter.$_extensionMethodName';
 /// eventually completes to a string response.
 typedef DataHandler = Future<String> Function(String message);
 
-class _DriverBinding extends BindingBase with ServicesBinding, SchedulerBinding, GestureBinding, PaintingBinding, SemanticsBinding, RendererBinding, WidgetsBinding {
+class _DriverBinding implements ServiceExtensionBinding {
```

Comment:
Same here. Once you change it, this is not a 'driver binding' anymore. It's just one service extension.